plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -243,9 +243,9 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 			dxd->Release();
 		}
 
-		bool spritehack_enabled = theApp.GetConfigB("UserHacks") && theApp.GetConfigI("UserHacks_SpriteHack");
+		bool nvidiahack_enabled = theApp.GetConfigB("UserHacks") && theApp.GetConfigB("UserHacks_Disable_Safe_Features");
 
-		m_hack_topleft_offset = (!nvidia_gpu || m_upscale_multiplier == 1 || spritehack_enabled) ? 0.0f : -0.01f;
+		m_hack_topleft_offset = (!nvidia_gpu || m_upscale_multiplier == 1 || nvidiahack_enabled) ? 0.0f : -0.01f;
```

Comment:
It's probably better to move this code up (or at least NV detection part) so you don't have to query the api for the adapter description twice.